cpp/lazperf/lazperf_user_base.hpp

```diff
@@ -0,0 +1,17 @@
+
+#pragma once
+
+#define LAZPERF_MAJOR_VERSION 1
+#define LAZPERF_MINOR_VERSION 3
+#define LAZPERF_REVISION 0
+#define LAZPERF_VERSION 1.3.0
+
+// Note that this __declspec(dllimport) is not necessary and you can remove it if you like.
+// It allows the compiler to make a potential optimization in calling exported functions
+// by using an indirect call rather than a call to a thunk which then calls the exported function.
+#ifdef _WIN32
+#define LAZPERF_EXPORT __declspec(dllimport)
+#else
+#define LAZPERF_EXPORT
+#endif
+
```

Comment:
This technic is quite unusual (split export macro in 2 files, leading to duplicated code), and don't forget that your test & example executables should also see `__declspec(dllimport)`, not just consumers after installation of laz-perf. The best way to handle export macro with CMake is https://cmake.org/cmake/help/v3.1/module/GenerateExportHeader.html
It's true that `__declspec(dllimport)` is not very important, except when you have global variables.